mm/2s2h/CustomMessage/CustomMessage.cpp

```diff
@@ -0,0 +1,64 @@
+
+#include "CustomMessage.h"
+#include <map>
```

Comment:
Including libultra here should fix the linux linker errors

```suggestion
#include <libultraship/bridge.h>
#include <map>
```